regexes/client/browsers.yml

```diff
@@ -1261,7 +1261,7 @@
   version: ''
 
 # Opera GX (https://www.opera.com/gx#start | http://ftp.opera.com/pub/opera_gx/)
-- regex: 'OPRGX(?:/(\d+[\.\d]+))?'
+- regex: 'OP(?:RG)?X(?:/(\d+[\.\d]+))?'
```

Comment:
https://apps.apple.com/en/app/opera-gx/id1559740799

* Mozilla/5.0 (iPhone; CPU iPhone OS 14_4_1 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Mobile/15E148 OPX/1.3.1

* Mozilla/5.0 (iPad; CPU OS 15_1 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Mobile/15E148 OPX/1.3.1

What type of engine will we get if it's an Iphone and iPad

Reply:
Blink

Reply:
Modify current rule to
```yml 
regex: 'Chrome/.+ OP(?:RG)?X(?:/(\d+[\.\d]+))?'
# ...
   engine: Blink
```
Create next rule
```yml

regex:  'OP(?:RG)?X(?:/(\d+[\.\d]+))?'
# ...
   engine: WebKit
 ```
And added test for IOS~
* Mozilla/5.0 (iPad; CPU OS 15_1 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Mobile/15E148 OPX/1.3.1